fix 2527, customize page name (#2850)
mui/mui-toolpad: docs/data/toolpad/concepts/page-properties.md

> - **App shell**: Pages with their display mode set to this value will render within the navigation sidebar on the left, allowing for easy navigation between all pages:

Reviewer: ```suggestion
- **App shell**: Pages with their display mode set to this value will render within the navigation sidebar on the left, allowing for easy navigation between all pages.
```

> - **App shell**: Pages with their display mode set to this value will render within the navigation sidebar on the left, allowing for easy navigation between all pages:
> 
> To display a different name for a page in the sidebar, the page `display name` property can be used.

Reviewer: ```suggestion
- **App shell**: Pages with their display mode set to this value will render within the navigation sidebar on the left, allowing for easy navigation between all pages. To display a different name for a page in the sidebar, the page `display name` property can be used.
```

> To display a different name for a page in the sidebar, the page `display name` property can be used.

Reviewer: ```suggestion
To display a different name for a page in the sidebar, the page **display name** can be set.
```

Just adjusting my own previous suggestion :)

```

Now the urls **/pages/workers** and **/pages/colleagues** will both redirect to the url **/pages/employees**.

## Page display name

> names of Toolpad objects must be valid javascript identifiers because they will create global variables in binding scripts. so can't use special characters in names. To display a different name for a page in the navigation sidebar, you can use the `displayName` property in the page's **page.yml** file. e.g.

Reviewer: ```suggestion
To display a different name for a page in the sidebar, the page `display name` can be used.
```

```yaml
# /pages/employees/page.yml
apiVersion: v1
kind: page
spec:
  displayName: 'displayName'

> ```

Reviewer: Thanks! We can move this text to just under the `App shell:` option, in a new paragraph above the image, and to keep it short remove the yaml example and change the text as suggested.
